public_html/index.html

```diff
@@ -35,6 +35,17 @@
         <h4 class="section-title">Top Stories</h4>
         <ul>
           <li class="highlight__item">
+            <a href="https://hacks.mozilla.org/2017/01/webgl-2-lands-in-firefox/" class="link">
+              <div class="imageWrapper">
+                <img src="assets/img/tile-webgl2.png">
```

Comment:
by running `pngquant` on this PNG, I was able to get the size down from `559 KB` to `168 KB`. the compression's quite good; I can barely discern any difference.